### ISSUE TITLE
 [dynamic import] import is not defined

### ISSUE THREAD

**Issue by ynwd** [denoland/deno_lint#339]
Typescript:
```ts
const service = await import(fileImport);
```

Error:
```
(no-undef) import is not defined
          const service = await import(fileImport);
                                ^^^^^^
    at /home/x230/me/app/server.ts:60:32
```

**Comment by kitsonk** [comment]
Duplicate of #321

**Comment by ynwd** [comment]
@kitsonk what is the solution of this error?

**Comment by lucacasonato** [comment]
Fix will land in Deno 1.4.3
